layouts/partials/scripts.html

```diff
@@ -0,0 +1,28 @@
+
+<script src="/js/popper-1.14.3.min.js" integrity="sha384-ZMP7rVo3mIykV+2+9J3UJ46jBk0WLaUAdn689aCwoqbBJiSnjAK/l8WvCWPIPm49" crossorigin="anonymous"></script>
+<script src="/js/bootstrap-4.3.1.min.js" integrity="sha384-JjSmVgyd0p3pXB1rRibZUAYoIIy6OrQ6VrjIEaFf/nJGzIxFDsf4x0xIM+B07jRM" crossorigin="anonymous"></script>
+
+{{ if .Site.Params.mermaid.enable }}
+<script src="/js//mermaid.min.js" crossorigin="anonymous"></script>
```

Comment:
nit:
```suggestion
<script src="/js/mermaid.min.js" crossorigin="anonymous"></script>
```

(but it's just a nit - the code as committed works OK)